.homeycompose/capabilities/measure_heater.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "boolean",
```

Comment:
Sorry, just had a maintenance power outage for the spa. Found out that this should be enum with values: ON, OFF or WAITING.

Reply:
Nice will change!